### ISSUE TITLE
System/system-processes-memory.rule and System/system-processes-cpu.rule don't filter by process name

### ISSUE THREAD

**Issue by lmakows-uva** [Juniper/healthbot-rules#583]
in the rules:

System/system-processes-memory.rule and System/system-processes-cpu.rule 

variable "process-name" is not used for input sensor filtering

**Comment by vvikramb** [comment]
You are right but we are filtering in trigger because HB 1.0 did not support filtering at field level for iagent sensors. Now HB supports filtering at field level for iagent sensors. We will incorporate changes in the next release.
